### ISSUE TITLE
No EddyStoneFramework or Socket-IO framework

### ISSUE THREAD

**Issue by JoeyBodnar** [Beaconstac/NearBee-iOS-SDK#2]
For manual installation, it says:

    Add the NearBee.framework, EddystoneScanner.framework, and Socket.IO-Client-Swift.framework 
    to the embedded binaries section of your destination app target.

However, when downloading the zip file for NearBee, there is no EddyStoneScanner.framework and no Socket.IO-Client-Swift.framework, and the installation doesn't say how I can find these. When trying to build my project, I get "module EddystoneScanner not found".

I went to the EddyStoneScanner GitHub (https://github.com/Beaconstac/EddystoneScanner-iOS-SDK) and there is no EddyStoneScanner.framework file there either when I downloaded the zip file.

Where can I find EddyStoneScanner.framework and Socket.IO-Client-Swift.framework? Thank you.

**Comment by sachinmobstac** [comment]
Please download the dependencies [Uploading NearBee_Dependencies.zip…](here).

**Comment by sachinmobstac** [comment]
@JoeyBodnar , Please download the dependencies from [here](UploadingNearBee_Dependencies.zip…).

**Comment by JoeyBodnar** [comment]
Hmm even after doing that I still got "Module EddystoneScanner not found":
    
<img width="657" alt="Screen Shot 2019-03-28 at 11 52 47 AM" src="https://user-images.githubusercontent.com/16038202/55131148-04b95880-5150-11e9-8f1b-7d1b4c087d5d.png">

So, now I am trying with Cocoapods and that is not working either. It compiles, but crashes on launch with this error message: 

<img width="652" alt="libraryNotLoaded" src="https://user-images.githubusercontent.com/16038202/55131415-1d763e00-5151-11e9-88d8-289efb535274.png">

Here is with my Linked Frameworks and Libraries looks like:

<img width="735" alt="Screen Shot 2019-03-28 at 12 03 56 PM" src="https://user-images.githubusercontent.com/16038202/55131534-92497800-5151-11e9-8bff-f6b5626b5d2c.png">

I have tried removing the "EddyStoneScanner", also tried making it optional, making it required, and tried making libPods-SauceGo.a both optional and required. In Build Settings, I have tried setting "Always Embed Swift Standard Libraries" to YES, and also tried setting it to NO. I have tried setting "Always Search User Paths" to both YES and NO as well.

Also tried: Under Build Phases -> Copy Bundle Resources, I tried adding libPods-SauceGo.a. 

However, no matter what I do, I always get the same error message where it says "library not loaded."

Xcode version: 10.1
Swift version: 4.2.1
My Podfile:
  
    platform :ios, '10.0'
      target 'MyTargetIsHere' do
        pod 'NearBee'
    end

cocoa pods version: 1.6.1 (I just installed cocoa pods yesterday)

Any help is appreciated, thank you.

**Comment by JoeyBodnar** [comment]
Continuing to try by manual installation as well. Now I have it compiling upon manual installation, but it crashes with a very similar error on launch:

  
<img width="747" alt="Screen Shot 2019-03-28 at 1 43 31 PM" src="https://user-images.githubusercontent.com/16038202/55135895-7b118700-515f-11e9-9b14-42d8f12751f5.png">

Also tried setting "Always Embed Swift Standard Libraries" under Build Settings to YES, and it still crashes.

If it is of any help, this is what my project directory looks like:

<img width="246" alt="Screen Shot 2019-03-28 at 1 47 22 PM" src="https://user-images.githubusercontent.com/16038202/55136084-0559eb00-5160-11e9-9c79-95920b9bd36d.png">

Thanks

**Comment by sachinmobstac** [comment]
@JoeyBodnar, Looking into it. Give me few minutes will update.

**Comment by sachinmobstac** [comment]
@JoeyBodnar, Missed few files in the frameworks have updated the [link] (https://drive.google.com/file/d/1-zPdUY-HI_guj8eDaTI74KsdBF2ASP7b/view). Please re-download the dependencies and try again.

**Comment by JoeyBodnar** [comment]
thank you. I just re-downloaded the dependencies for manual installation, but I am still getting a crash with the same error message. When I opened the .framework contents though, the file and everything looked exactly the same. Maybe the link wasn't updated?

thanks for you help

**Comment by sachinmobstac** [comment]
@JoeyBodnar, Can I get access to your project. I started a new Swift Project, it works just fine. I am not sure what's the issue with your Project.

**Comment by JoeyBodnar** [comment]
For legal reasons I'm not sure if I can let you see the project, sorry. But I did just start a brand new Xcode project and I am having the same problem. Are you running on a simulator or the device? I am trying to run on the simulator. I searched part of the error message "no matching architecture in universal wrapper" and it looks like maybe NearBee cannot be built using the simulator? https://forums.developer.apple.com/thread/11237

It may be that trying to use the simulator is my problem. At the moment I can't run on a device because my Xcode is 10.1 and I already updated my phone's iOS to 12.2, and they are incompatible.

**Comment by sachinmobstac** [comment]
@JoeyBodnar, Since NearBee depends extensively on the Bluetooth to scan and show the data related to beacon we have disabled the Simulator architecture. This is because, the Simulator cannot use the MacBook's BLE and will never be able to see it working.

**Comment by JoeyBodnar** [comment]
ahhh ok I see. this is probably the problem. Ok I will get Xcode 10.2 and see if I can build with that. Thanks a lot!

**Comment by JoeyBodnar** [comment]
Ok so I have obtained an iPhone with iOS 12.1 and am building on Xcode 10.1. I am still have the same same problem with manual installation. I created a brand new Xcode project that has this problem, here: https://github.com/JoeyBodnar/TestNearBee. I am using manual installation

It compiles, unless you add "import NearBee". When adding the import statement then it does not compile and I receive the following errors:

<img width="709" alt="Screen Shot 2019-03-29 at 2 50 28 PM" src="https://user-images.githubusercontent.com/16038202/55217384-ffccd580-5231-11e9-9865-f8c641f3434b.png">

What I have tried:
- I have deleted Derived Data and gone to Product -> Clean Build folder at least 5 times. 
- Restarted Xcode
- Went to Build Phases -> Target Dependencies, but none of them were visible to add
- Build Settings -> Always Embed Swift Standard Libraries, tried YES and NO
